src/index.js

```diff
@@ -0,0 +1,54 @@
+import './style.css';
+import moreIcon from './more.svg';
+import reload from './reload.svg';
+import enter from './enter.svg';
+
+const ToDoListContainer = document.getElementById('list-container');
+const reloadIcon = document.getElementById('reload-img');
+const enterIcon = document.getElementById('enter-icon');
+enterIcon.src = enter;
+reloadIcon.src = reload;
+const todoList = [
+  {
+    id: 1,
+    description: 'Starting project structure',
+    completed: false,
+  },
```

Comment:
- [ ] As suggested by the previous reviewer, kindly consider adhering to the project requirements that state that each task should have an `index` instead of an `id`. Please update your task list to have an `index`. See below for more details on project requirements:
<img width="981" alt="Screenshot 2023-07-22 at 08 14 06" src="https://github.com/AndresEspi/toDoList/assets/107192076/956a2445-21da-4355-b894-b088a3e61fc8">